Makefile

```diff
@@ -17,28 +20,26 @@ testall: fennel
 	@printf "\nTesting lua 5.4:\n"; lua5.4 test/init.lua
 	@printf "\nTesting luajit:\n" ; luajit test/init.lua
 
-luacheck:
-	luacheck fennel.lua test/init.lua test/mangling.lua \
-		test/misc.lua test/quoting.lua
-
 count:
-	cloc fennel.lua
-	cloc --force-lang=lisp fennelview.fnl fennelfriend.fnl fennelbinary.fnl \
-		launcher.fnl
+	cloc --force-lang=lisp $(SRC) # core compiler
+	cloc --force-lang=lisp $(EXTRA_SRC) # libraries and launcher
 
-# For the time being, avoid chicken/egg situation thru the old Lua launcher.
-LAUNCHER=$(LUA) old_launcher.lua
+# Avoid chicken/egg situation using the old Lua launcher.
+LAUNCHER=$(LUA) old/launcher.lua --add-fennel-path src/?.fnl --globals "_G,_ENV"
 
 # Precompile fennel libraries
-%.lua: %.fnl fennel.lua
-	 $(LAUNCHER) --globals "" --compile $< > $@
+fennelview.lua: fennelview.fnl fennel.lua ; $(LAUNCHER) --compile $< > $@
+fennelfriend.lua: fennelfriend.fnl fennel.lua ; $(LAUNCHER) --compile $< > $@
```

Comment:
Should we continue to keep these generated files in the repo, or should we move them to the downloads section of the web site? (We can address this later on.)

Reply:
Agree we should address it later on, but I think we should move to an all-or-nothing approach - either we leave all the generated files out, or check them all in and mark them as generated in `.gitattributes`.

If we do the latter, we should also add a CI check to ensure that at the very least all the timestamps match, either that or do a checksum of the built files in the repo against some fresh-built equivalents.

Reply:
Since a CI run will remake the generated files, maybe we can add a final step to the CI which fails the build if the generated files have changed according to git.

Reply:
True, we could just add something like `git diff --quiet`, which will exit with a `1` status if there are changed files. Or there's probably a circleci orb for it, but I'm not super familiar with that ecosystem.

Reply:
Cool; added in aaa9f8d.